go-controller/pkg/node/gateway_shared_intf_linux.go

```diff
@@ -38,7 +38,15 @@ func setupLocalNodeAccessBridge(nodeName string, subnets []*net.IPNet) error {
 		return err
 	}
 
-	macAddress := util.IPAddrToHWAddr(net.ParseIP(util.V4NodeLocalNatSubnetNextHop)).String()
+	var macAddress string
+	if config.IPv4Mode && config.IPv6Mode {
+		macAddress = util.IPAddrToHWAddr(net.ParseIP(util.V4NodeLocalNatSubnetNextHop)).String()
```

Comment:
why in dual stack we default always to ipv4?

Reply:
That's the way it is done here:
https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/gateway_init.go#L413

Reply:
I can't grok entirely the comment on top https://github.com/ovn-org/ovn-kubernetes/blob/3ce369550d68eff8e37cd021c16c823b823ae756/go-controller/pkg/ovn/gateway_init.go#L403-L408 
and I don't have enough knowledge to judge, but at least I think we should left a comment explaining the rationale or pointing to the link you left

Reply:
The interface can only have a single MAC address. By convention, in dual-stack mode we use the MAC computed from the IPv4 address rather than the one computed from the IPv6 address. Originally this was because the MAC-generation code used bad heuristics under IPv6, but that's been fixed now, so we _could_ use either. But we have to use one or the other, and the one we use is the IPv4 one.